tests/kuttl/patch-object.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -eo pipefail
+set -x
+
+user_type=${1}
+bucket_type=${2}
+
+user_name=$(kubectl get ${user_type} -oname)
+bucket_name=$(kubectl get ${bucket_type} -oname)
```

Comment:
I generally recommend to put a space between the option and the argument value for easier readability.
```suggestion
user_name=$(kubectl get ${user_type} -o name)
bucket_name=$(kubectl get ${bucket_type} -o name)
```